.github/workflows/build.yml

```diff
@@ -409,35 +409,20 @@ jobs:
           distribution: 'zulu'
           java-version: 11
 
-      - name: Gradle cache
-        uses: gradle/gradle-build-action@v2
-
-      - name: AVD cache
-        uses: actions/cache@v3
-        id: avd-cache
-        with:
-          path: |
-            ~/.android/avd/*
-            ~/.android/adb*
-          key: avd-${{ matrix.api-level }}
-
-      - name: Create AVD and generate snapshot for caching
-        if: steps.avd-cache.outputs.cache-hit != 'true'
-        uses: reactivecircus/android-emulator-runner@v2
-        with:
-          api-level: ${{ matrix.api-level }}
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          script: echo "Generated AVD snapshot for caching."
+      - name: Accept licenses
+        run: |
+          mkdir -p /Users/runner/Library/Android/sdk
+          tar xf .github/android_licenses.tar.gz -C /Users/runner/Library/Android/sdk
```

Comment:
This song and dance works for me, and it’s also bizarre.

Who is accepting these licenses? Square, Inc.? Does accepting it both mechanically and publicly obligate us to anything?

Reply:
I asked some internal people if we have Opinions on stuff like this.

Reply:
When you run various gradle plugins, it writes these files, it just doesn't know about previews

Reply:
I guess I should be using a step like the following with channel set.  Will test.

```
yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager --licenses || true
```